crypto/sign_test.go

```diff
@@ -92,7 +84,8 @@ func TestSignAndVerifyMessageEthereum(t *testing.T) {
 	fmt.Println("signature:", hexutil.Encode(signature))
 	fmt.Println("Generated Signature can also be verified at https://etherscan.io/verifySig")
 
-	correct := VerifyMessage(publicKey, testMsg, signature, CurveSecp256K1, true)
+	pk32 := utils.AddressTo32Bytes(common.HexToAddress(address))
+	correct := VerifyMessage(pk32[:], testMsg, signature, CurveSecp256K1)
```

Comment:
I think there are a bunch of redundant tests since we moved to default eth prefix for secp256k. Can you remove those as well?